src/components/SelectedAnnotation.jsx

```diff
@@ -185,6 +200,14 @@ class SelectedAnnotation extends React.Component {
     );
   }
 
+  cancelAnnotation() {
+    if (this.props.onClose) { this.props.onClose() };
+
+    if (this.context.googleLogger) {
+      this.context.googleLogger.logEvent({ type: 'cancel-transcription' });
```

Comment:
Worth checking: does the cancel-transcription logging need to happen _after_ props.onClose() ? I'm not sure if onClose() kills the component and prevents the log from triggering. Probably not - the code block should run properly, but we could do a quick double check.

Reply:
I confirmed this a few times and it appears the call is made in the network tab before the modal closes.